packages/katello/rubygem-hammer_cli_foreman_scc_manager/rubygem-hammer_cli_foreman_scc_manager.spec

```diff
@@ -20,15 +20,13 @@ Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 Autoreq: 0
```

Comment:
This needs to be dropped since the current generated dependencies counts on autorequires